dev/MRTCore/packaging/Microsoft.ApplicationModel.Resources.PriGen.targets

```diff
@@ -2225,8 +2177,6 @@
       </OptionalProjectOutputs>
     </ItemGroup>
 
-    <Message Text="$(ProjectName) : %(OptionalProjectOutputs.ProjectName).%(OptionalProjectOutputs.OutputGroup) - %(OptionalProjectOutputs.FileType) : %(OptionalProjectOutputs.Identity) -> %(OptionalProjectOutputs.TargetPath)" />
```

Comment:
This change, and one above it, are key for perf improvements, as they are repeating thousands of times for large projects...